### ISSUE TITLE
Older IndexedDB not compatible with 7.1.0

### ISSUE THREAD

**Issue by jonibahu** [deanoemcke/thegreatsuspender#1002]
google chrome
version 7.1.0 (and 6.40 before update)
windows 8

The extension updated to 7.1.0
My session were still there.
I pressed recover and the extension got corrupted, so i repaired it.
After that my sessions were gone.

I recovered an older IndexedDB file.
But everytime i put that into the folder, it gets automatically deleted.

Are my recovered IndexedDB files corrupted, or will the files from v6.40 not work with 7.1.0 anymore?????
I would really appriciate an answer.
https://github.com/deanoemcke/thegreatsuspender/issues/817 i also did everything suggested in this post to make it work but it still did not read out the saved sessions :(

**Comment by liamjohnston** [comment]
The bad news is there is a bug in the import code. The good news is that you can install [Session Buddy](https://chrome.google.com/webstore/detail/session-buddy/edacconmaakjimmfgnblocblbcdcpbko?hl=en) and import a Great Suspender session successfully. I hope this helps :)

**Comment by jonibahu** [comment]
Hi
I can import without any issues (it works for sessions i exported in a .txt file)


The issue is, i had a few sessions saved in the session manager.
Those are gone after the extension got corrupted.
I wanted to restore them in some way.

I tried recovering the IndexedDB file, but the extension always deletes those files when i put them into the current indexedDB.

(I tried making new session saves, deleting those by deleting the indexedDB worked, and when i put that indexedDB back in the new saved sessions showed back up)
So it should work in theory, but for some reason it doesnt with my older files.



I even tried installing v 6.30 from github, but even that could not read my IndexedDB files. (still deleted the older version of them everytime i put them in)

So im not sure what exactly causes this, i THINK the older files should not be broken, but maybe they are.
for some reason they always get deleted tho when i try to paste them into the indexedDB directory.

here are my older indexedDB files: https://www.file-upload.net/download-13702680/111.rar.html

My theory is they are alright but i gotta do something else for them to be read properly.
But maybe they are corrupted.

If u could help me tell it would go along way.
